### ISSUE TITLE
Lua error (calling a protected function during combat lockdown)

### ISSUE THREAD

**Issue by timzwiebel** [arkayenro/arkinventory#1777]
I was trying to move an item in my inventory (while in combat) and got a lua error. Now the item is grayed out and I can't move it until I log out and back in again (a UI reload does not fix the issue).

Date: 2023-02-11 14:24:05
ID: 1
Error occured in: AddOn: ArkInventory
Count: 1
Message: Note: AddOn ArkInventory attempted to call a protected function (PickupContainerItem()) during combat lockdown.
Debug:
   [string "=[C]"]: PickupContainerItem()
   [string "@Interface/FrameXML/ContainerFrame_Shared.lua"]:1256: ContainerFrameItemButton_OnClick()
   [string "@Interface/AddOns/ArkInventory/Core/ArkInventory.lua"]:9425: Frame_Item_OnReceiveDrag()
   [string "*ArkInventory.xml:496_OnReceiveDrag"]:1:
      [string "*ArkInventory.xml:496_OnReceiveDrag"]:1

**Comment by arkayenro** [comment]
which game client?

i cant duplicate this in retail.  i can use and move items around in combat.

**Comment by timzwiebel** [comment]
WotLK Classic. (BlizRuntimeLib_enUS v3.4.1.30401)

I also can usually move items while in combat. But I've run into this issue twice, so I figured I'd report it.

**Comment by arkayenro** [comment]
i can duplicate it in wrath but its not something i can fix.

its never been blocked before so i presume its a blizzard screwup - as it means no bag mod would ever work while in combat.

i do get the error but stuff still moves.

**Comment by timzwiebel** [comment]
Were you able to find a way to reproduce the issue reliably in WotLK Classic? It only occurs rarely for me.

**Comment by arkayenro** [comment]
yes, i just walked over a level 2 mob to get aggro then opened the bag and moved something around and it barfed, although as i said, it kept working after that (which is weird for a blocked warning)

i dont have much time at the moment but if i get the chance i might enable taint logging just to double check its not something else.
